composer.json

```
@@ -22,8 +22,9 @@
         }
     ],
     "require": {
-        "php": ">=5.6.0",
-        "psr/log": "^1.0"
+        "php": ">=8.0",
```

Aura 5.x requires PHP 8.1 or later.

```suggestion
        "php": ">=8.1",
```

---

It does work fine on php 8.0, but I am fine with it being set to 8.1, did mean to remove the psr/log requirement?

---

Aura 5.x does not support PHP 8.0.
PHP 8.0 is supported by Aura 4.x or 3.x.

I did not mean remove the psr/log.